test/image/mocks/bar_autorange-above-zero-normalized.json

```diff
@@ -0,0 +1,19 @@
+{
+  "data": [
+    {
+      "y": [1, 2, 3],
+      "base": [3, 4, 5],
+      "type": "bar"
+    },
+    {
+      "y": [1, 2, 3],
+      "base": [3, 4, 5],
+      "type": "bar"
+    }
+  ],
+  "layout": {
+    "barnorm": "percent",
+    "width": 400,
+    "height": 400
```

Comment:
It seems to me that the previous bug only happened with positive numbers?
If that's not the case, it would be great if you could add another subplot using negative numbers or reversed autorange.

Reply:
I'm pretty sure our current test coverage is good enough. Negating the `tozero` values passed to `Axes.findExtremes` from https://github.com/plotly/plotly.js/pull/3452/commits/7e0a22895449cdf553106dfa1a89480eddf05be4 makes 34 image test fail.